src/pages/Dashboard/Modals/SupplyWithdraw/index.tsx

```diff
@@ -92,69 +123,81 @@ export const SupplyWithdrawUi: React.FC<ISupplyWithdrawUiProps> = ({
-                  [
-                    <SupplyWithdrawForm
-                      key="supply"
-                      asset={asset}
-                      tokenInfo={tokenInfo}
-                      userTotalBorrowBalance={userTotalBorrowBalance}
-                      userTotalBorrowLimit={userTotalBorrowLimit}
-                      dailyEarnings={dailyEarnings}
-                      onSubmit={onSubmit}
-                      inputLabel={t('supplyWithdraw.walletBalance')}
-                      enabledButtonKey={t('supplyWithdraw.supply')}
-                      disabledButtonKey={t('supplyWithdraw.enterValidAmountSupply')}
-                      maxInput={asset.walletBalance}
-                      calculateNewBalance={calculateNewSupplyAmount}
-                    />,
-                    <SupplyWithdrawForm
-                      key="withdraw"
-                      asset={asset}
-                      tokenInfo={tokenInfo}
-                      userTotalBorrowBalance={userTotalBorrowBalance}
-                      userTotalBorrowLimit={userTotalBorrowLimit}
-                      dailyEarnings={dailyEarnings}
-                      onSubmit={onSubmit}
-                      inputLabel={t('supplyWithdraw.withdrawableAmount')}
-                      enabledButtonKey={t('supplyWithdraw.withdraw')}
-                      disabledButtonKey={t('supplyWithdraw.enterValidAmountWithdraw')}
-                      maxInput={asset.supplyBalance}
-                      calculateNewBalance={calculateNewBorrowAmount}
-                    />,
-                  ][activeTabIndex]
-                }
-              </EnableToken>
-            )}
+            <EnableToken
+              symbol={assetId as TokenId}
+              title={activeTabTextKeys.enable}
+              tokenInfo={tokenInfo}
+              isEnabled={!!isEnabled}
+              vtokenAddress={asset.vtokenAddress}
+            >
+              {
+                [
+                  <SupplyWithdrawForm
+                    key="supply"
+                    asset={asset}
+                    tokenInfo={tokenInfo}
+                    userTotalBorrowBalance={userTotalBorrowBalance}
+                    userTotalBorrowLimit={userTotalBorrowLimit}
+                    dailyEarnings={dailyEarnings}
+                    onSubmit={onSubmitSupply}
+                    inputLabel={t('supplyWithdraw.walletBalance')}
+                    enabledButtonKey={t('supplyWithdraw.supply')}
+                    disabledButtonKey={t('supplyWithdraw.enterValidAmountSupply')}
+                    maxInput={asset.walletBalance}
+                    calculateNewBalance={calculateNewSupplyAmount}
+                    isTransactionLoading={isTransactionLoading}
+                  />,
+                  <SupplyWithdrawForm
+                    key="withdraw"
+                    asset={asset}
+                    tokenInfo={tokenInfo}
+                    userTotalBorrowBalance={userTotalBorrowBalance}
```

Comment:
I caught this just now, but here we'll need to pass another prop. Currently, the withdraw form displays the total borrow balance of the user beneath the input, but it should display the amount of tokens the user have supplied instead.